modules/rtdModule/index.js

```diff
@@ -196,11 +222,12 @@ export function requestBidsHook(fn, reqBidsConfigObj) {
  * @param {Object} data - key values to set
  */
 function setDataForPrimaryAdServer(data) {
-  if (!utils.isGptPubadsDefined()) {
-    utils.logError('window.googletag is not defined on the page');
-    return;
-  }
-  targeting.setTargetingForGPT(data, null);
+  data = validateProviderDataForGPT(data);
+  utils.isGptPubadsDefined()
+    ? targeting.setTargetingForGPT(data, null)
+    : window.googletag.cmd.push(() => {
+      targeting.setTargetingForGPT(data, null);
```

Comment:
👍 For pushing on the googletag cmd que, but should we check to make sure all of it <window.googletag.cmd> is defined before trying to execute the push though?

If either `googletag` or `cmd` is not defined by the time this runs it will throw an error I believe.

Reply:
Yes, you are right
I added googletag init

Reply:
Nice!